core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -35,6 +35,7 @@ var ExplorationEditorMainTab = function() {
     by.css('.protractor-test-add-response-details'));
   var addResponseHeader = element(
     by.css('.protractor-test-add-response-modal-header'));
+  var getAnswerTabs = element.all(by.css('.protractor-test-answer-tab'));
```

Comment:
The other variables here are named as nouns; why is this a verb?

(/cc @U8NWXD -- is this naming convention expected? Just wanted to check.)

Reply:
Thanks for catching this @seanlip ! Yes, this should be named as a noun. In general @donosco98, functions should be named as verbs, and variables should be named as nouns.